resources/tentative/complex-static-html/src/big-dom-generator/gen-css.js

```diff
@@ -0,0 +1,231 @@
+import { LCG } from 'random-seedable';
+import { SEED, MAX_SELECTOR_LEN } from './params.js';
+
+const rando = new LCG(SEED);
```

Comment:
Please don't use abbreviations like rando. Spell out the full word: random.

Reply:
Done.